src/components/molecules/HistoricalChart/HistoricalChart.tsx

```diff
@@ -118,14 +117,6 @@ export function HistoricalChart({
                 ...volumeData,
             } as Omit<ITradingData, 'vol'> & { value: number };
 
-            const date = new Date(Number(candleData?.time) * 1000);
```

Comment:
looks like unrelated to tabs/sub tabs. Could you give some background for this change?

Reply:
@as472 Yeah these are unrelated to tabs/subtabs. I see that the width for Historical Chart is smaller on small desktop screens, and that @benk-sf removed the hover time display on the chart: https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6/Product-UI-(Main)?node-id=15841-65963&m=dev

On our current implementation, the hover time/date is being cut off because of insufficient space, so I thought this would be a good time to remove that